issuer-front-end/src/SelectNetwork.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+import React, { Dispatch, SetStateAction } from 'react';
+
+interface ConnectionProps {
+    setIsTestnet: Dispatch<SetStateAction<boolean | undefined>>;
+}
+
+export default function SelectNetwork(props: ConnectionProps) {
+    const { setIsTestnet } = props;
+
+    const changeDropDownHandler = () => {
+        const e = document.getElementById('write') as HTMLSelectElement;
+        const sel = e.selectedIndex;
+        const { value } = e.options[sel];
+        if (value === 'Testnet') {
+            setIsTestnet(true);
+        } else if (value === 'Mainnet') {
+            setIsTestnet(false);
+        } else {
+            console.error('Select a network');
+            return;
+        }
+
+        const progressNext = document.getElementById('progress-next') as HTMLTextAreaElement;
+        progressNext.disabled = false;
```

Comment:
This seems a bit dirty 😜 Should set some state in a top-level component, which the corresponding JSX element reads instead.